.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: go
-script: go test -race -cpu 1,2,4 -v -timeout 2m ./...
+script: ETCDTESTS=1 go test -race -cpu 1,2,4 -v -timeout 5m ./...
```

Comment:
I've heard there's a way, but it's definitely not obvious. I couldn't find docs on it.